airflow/models/all_models/__init__.py

```diff
@@ -0,0 +1,154 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# flake8: noqa: F401
+# pylint: disable=redefined-outer-name, too-many-locals, too-many-return-statements, too-many-branches
```

Comment:
I think you misunderstood me - so maybe i explain. The only reason we have the "all_models" is so that resetdb can import all the models (so all classes deriving from sqlalchemy's 'Base'
class. And all_models will only be used there, so there is no need to lazy load (actually there is a need to load them immediately when `from airflow.models.all_models  import *` is used